proprietary/tokens/src/components/ams/ordered-list.tokens.json

```
@@ -25,6 +25,7 @@
       },
       "ordered-list": {
         "list-style-type": { "value": "lower-alpha" },
+        "padding-block-start": { "value": "{ams.space.xs}" },
```

The designers do think `sm` is better here.

```suggestion
        "padding-block-start": { "value": "{ams.space.sm}" },
```